README.md

```diff
@@ -94,16 +94,11 @@ and the data files in `/tmp/share`.
 
  ## The game
  
- The game takes place on a rectangular arena where a snake continuously
- move in one of the four directions: left, right, up and down --- it never 
- stops. As the snake moves it looses energy and if all of it is exausted, the
- snake dies. To recover energy, the snake needs to eat pieces of food which
- are constantly replaced at random positions. 
+The game takes place in a rectangular arena where a snake continuously
+moves in one of four directions: left, right, up, and down --- it never 
+stops. As the snake moves, it loses energy and if all of it is exhausted, the snake dies. To recover energy, the snake needs to eat pieces of food that are constantly replaced at random positions.
```

Comment:
I think this had been purposely broken into different lines to make it easier to edit later (since single line breaks don't affect markdown output)
```suggestion
stops. As the snake moves, it loses energy and if all of it is exhausted, the
snake dies. To recover energy, the snake needs to eat pieces of food that are
constantly replaced at random positions.
```